uiowa/uiowa
# SEO Metatag Setting Adjustments

Resolves #2859

Expands scope to deal with other issues...

## To Test

- Advanced robots SEO settings should be available to webmasters on article, page, person. Not sure how to test this locally to see if it actually blocks or does what it says when checked, but the configuration should be visible to webmasters.
- Person SEO Settings should now be available to webmasters, publishers, editors
- Coronavirus should have no loss in permissions for webmaster/editor as it was identical to the default config.
- Area of Study feature split should be closer to default with regard to new blocks and updates to the metatag module.
- Saving the person form/display settings on grad, admissions, immuno.grad and person_extended sites should not change anything but the metatag field being there.
- Person hometown is captured in config but only available for use on Admissions.

## Todo

core.entity_view_display.node.person.teaser

- [x] config/grad.uiowa.edu/core.entity_view_display.node.person.teaser.yml
- [x] config/admissions.uiowa.edu/core.entity_view_display.node.person.teaser.yml
- [x] config/immuno.grad.uiowa.edu/core.entity_view_display.node.person.teaser.yml
- [x] config/features/person_extended/core.entity_view_display.node.person.teaser.yml

user.role.editor

- [x] config/hr.uiowa.edu/user.role.editor.yml
- [x] config/grad.uiowa.edu/user.role.editor.yml
- [x] config/brand.uiowa.edu/user.role.editor.yml
- [x] config/features/sitenow_v2/user.role.editor.yml
- [x] config/admissions.uiowa.edu/user.role.editor.yml
- [x] config/features/scholarship/user.role.editor.yml
- [x] ~~config/coronavirus.uiowa.edu/user.role.editor.yml~~ - _identical to default, removing_
- [x] config/policy.clas.uiowa.edu/user.role.editor.yml
- [x] config/features/areas_of_study/user.role.editor.yml
- [x] config/ighn.international.uiowa.edu/user.role.editor.yml

user.role.publisher

- [x] config/hr.uiowa.edu/user.role.publisher.yml
- [x] config/brand.uiowa.edu/user.role.publisher.yml
- [x] config/admissions.uiowa.edu/user.role.publisher.yml
- [x] config/features/scholarship/user.role.publisher.yml
- [x] config/policy.clas.uiowa.edu/user.role.publisher.yml
- [x] config/features/areas_of_study/user.role.publisher.yml
- [x] config/ighn.international.uiowa.edu/user.role.publisher.yml

user.role.webmaster

- [x] config/uiowa.edu/user.role.webmaster.yml
- [x] config/features/apr/user.role.webmaster.yml
- [x] config/hr.uiowa.edu/user.role.webmaster.yml
- [x] config/grad.uiowa.edu/user.role.webmaster.yml
- [x] config/brand.uiowa.edu/user.role.webmaster.yml
- [x] config/features/sitenow_v2/user.role.webmaster.yml
- [x] config/admissions.uiowa.edu/user.role.webmaster.yml
- [x] config/features/scholarship/user.role.webmaster.yml
- [x] ~~config/coronavirus.uiowa.edu/user.role.webmaster.yml~~ - _identical to default, removing_
- [x] config/policy.clas.uiowa.edu/user.role.webmaster.yml
- [x] config/features/areas_of_study/user.role.webmaster.yml
- [x] config/ighn.international.uiowa.edu/user.role.webmaster.yml
- [x] config/presidentialsearch.uiowa.edu/user.role.webmaster.yml

core.entity_view_display.node.person.default

- [x] config/grad.uiowa.edu/core.entity_view_display.node.person.default.yml
- [x] config/admissions.uiowa.edu/core.entity_view_display.node.person.default.yml
- [x] config/immuno.grad.uiowa.edu/core.entity_view_display.node.person.default.yml
- [x] config/features/person_extended/core.entity_view_display.node.person.default.yml


core.entity_form_display.node.person.minimal.yml

- [x] config/grad.uiowa.edu/core.entity_form_display.node.person.minimal.yml
- [x] config/admissions.uiowa.edu/core.entity_form_display.node.person.minimal.yml
- [x] config/immuno.grad.uiowa.edu/core.entity_form_display.node.person.minimal.yml
- [x] config/features/person_extended/core.entity_form_display.node.person.minimal.yml


core.entity_form_display.node.person.default.yml

- [x] config/grad.uiowa.edu/core.entity_form_display.node.person.default.yml
- [x] config/admissions.uiowa.edu/core.entity_form_display.node.person.default.yml
- [x] config/immuno.grad.uiowa.edu/core.entity_form_display.node.person.default.yml
- [x] config/features/person_extended/core.entity_form_display.node.person.default.yml
